redhat-cop/agnosticd
# Fix jupyterhub error in Data Engineering workshop

##### SUMMARY
Fixes error preventing the jupyter notebook from spawning successfully in the Data Engineering workshop. Updates the Open Data Hub deployment to 0.5.2

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
ocp4-workload-rhte-analytics_data_ocp_workshop

##### ADDITIONAL INFORMATION
VERIFICATION STEPS:
Deploy the Data Engineering workshop
Login as the user and attempt to spawn a jupyter notebook
the notebook should spawn without any errors displayed
